aiohttp/__init__.py

```diff
@@ -38,7 +38,11 @@
 )
 from .cookiejar import CookieJar as CookieJar, DummyCookieJar as DummyCookieJar
 from .formdata import FormData as FormData
-from .helpers import BasicAuth as BasicAuth, ChainMapProxy as ChainMapProxy
+from .helpers import (
+    BasicAuth as BasicAuth,
+    ChainMapProxy as ChainMapProxy,
+    ETag as ETag,
+)
```

Comment:
Why not simply `from .helpers import BasicAuth, ChainMapProxy, ETag`?

Reply:
I guess it's a kind of outer interface protection measures. Andrew can tell for sure.

Reply:
No, we have no policy for such cases.
Usually, we use `as` only for importing under a different name if there is a conflict.

Please feel free to drop `as`.

Reply:
@Dreamsorcerer